app/src/main/java/com/xmartlabs/scasas/doapp/ui/onboarding/OnBoardingActivity.java

```diff
@@ -0,0 +1,75 @@
+package com.xmartlabs.scasas.doapp.ui.onboarding;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.DrawableRes;
+import android.support.annotation.StringRes;
+import android.support.v4.view.ViewPager;
+
+import com.codemybrainsout.onboarder.AhoyOnboarderActivity;
+import com.codemybrainsout.onboarder.AhoyOnboarderCard;
+import com.f2prateek.dart.Dart;
+import com.f2prateek.dart.InjectExtra;
+import com.xmartlabs.scasas.doapp.R;
+import com.xmartlabs.scasas.doapp.model.User;
+import com.xmartlabs.scasas.doapp.ui.Henson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by scasas on 3/13/17.
+ */
+public class OnBoardingActivity extends AhoyOnboarderActivity {
+  public static final int PADDING_BOTTOM = 91;
```

Comment:
in the future take care about the unit! I think you are working with px, and you should work with dp